### PR TITLE
Powerng 54

### PR DESCRIPTION
**Pull request by miksik98**
Card Sorting Test's Results Visualization

### CODE REVIEW

File: src/app/components/view-results-of-test/view-results-of-test.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  ngOnInit(): void {
+  }
+
+  host = 'http://localhost:8099';
+  testsEndpoint = '/api/cardsorting/tests/';
+  testID : bigint;
+  processedResults = [];
+
+  getResults(){
```
Comment (Sroczek) [inline src/app/components/view-results-of-test/view-results-of-test.component.ts:25]: This method breaks SRP. View-results-of-test do retriew data from server and show it. Component should rather focus on presenting data and delegate data access to a service. You can read about this here: https://angular.io/tutorial/toh-pt4

File: src/app/components/view-results-of-test/view-results-of-test.component.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+export class ViewResultsOfTestComponent implements OnInit {
+
+  constructor(private titleService: Title, private http: HttpClient) {
+    this.titleService.setTitle('View Results of Card Sorting Test');
+  }
+
+  ngOnInit(): void {
+  }
+
+  host = 'http://localhost:8099';
```
Comment (Sroczek) [inline src/app/components/view-results-of-test/view-results-of-test.component.ts:20]: Use environment.ts - the host address is then easier to replace.

File: src/app/components/view-results-of-test/view-results-of-test.component.ts (hunk 1 above)
Comment (miksik98) [reply src/app/components/view-results-of-test/view-results-of-test.component.ts:23]: In the new commits I created service for Card Sorting and moved every function which uses http method to this service.

File: src/app/components/view-results-of-test/view-results-of-test.component.ts (hunk 2 above)
Comment (miksik98) [reply src/app/components/view-results-of-test/view-results-of-test.component.ts]: Ok, I set it in CardSortingService.

**Pull request merged**